source/features/diff-view-without-whitespace-option.tsx

```diff
@@ -40,6 +40,17 @@ function init() {
 	if (uselessCopy) {
 		uselessCopy.firstChild.remove();
 	}
+
+	// Remove whitespace settings in Diff Settings dropdown
+	const hideWhitespaceCheckbox = select('#whitespace-cb');
+	if (hideWhitespaceCheckbox) {
+		// Remove header
+		hideWhitespaceCheckbox.previousElementSibling.remove();
+		// Remove checkbox label
+		hideWhitespaceCheckbox.nextElementSibling.remove();
+		// Remove checkbox
+		hideWhitespaceCheckbox.remove();
```

Comment:
This is pretty ugly but it's safer this way 👍 

You can skip the `if` so if it's missing we'll get an error in the console